### ISSUE TITLE
install error with WinError 5

### ISSUE THREAD

**Issue by 5118Python** [OpenBMB/BMTrain#52]
(cpm) D:\GitHub\BMTrain>python setup.py install
running install
C:\ProgramData\Anaconda3\envs\cpm\lib\site-packages\setuptools\command\install.py:37: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  setuptools.SetuptoolsDeprecationWarning,
C:\ProgramData\Anaconda3\envs\cpm\lib\site-packages\setuptools\command\easy_install.py:147: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
  EasyInstallDeprecationWarning,
running bdist_egg
running egg_info
writing bmtrain.egg-info\PKG-INFO
writing dependency_links to bmtrain.egg-info\dependency_links.txt
writing requirements to bmtrain.egg-info\requires.txt
writing top-level names to bmtrain.egg-info\top_level.txt
reading manifest file 'bmtrain.egg-info\SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
writing manifest file 'bmtrain.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
creating build\lib.win-amd64-cpython-37
creating build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\block_layer.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\checkpointing.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\debug.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\global_var.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\init.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\layer.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\parameter.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\param_init.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\pipe_layer.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\store.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\synchronize.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\utils.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\wrapper.py -> build\lib.win-amd64-cpython-37\bmtrain
copying bmtrain\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain
creating build\lib.win-amd64-cpython-37\bmtrain\benchmark
copying bmtrain\benchmark\all_gather.py -> build\lib.win-amd64-cpython-37\bmtrain\benchmark
copying bmtrain\benchmark\reduce_scatter.py -> build\lib.win-amd64-cpython-37\bmtrain\benchmark
copying bmtrain\benchmark\send_recv.py -> build\lib.win-amd64-cpython-37\bmtrain\benchmark
copying bmtrain\benchmark\shape.py -> build\lib.win-amd64-cpython-37\bmtrain\benchmark
copying bmtrain\benchmark\utils.py -> build\lib.win-amd64-cpython-37\bmtrain\benchmark
copying bmtrain\benchmark\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain\benchmark
creating build\lib.win-amd64-cpython-37\bmtrain\distributed
copying bmtrain\distributed\ops.py -> build\lib.win-amd64-cpython-37\bmtrain\distributed
copying bmtrain\distributed\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain\distributed
creating build\lib.win-amd64-cpython-37\bmtrain\inspect
copying bmtrain\inspect\format.py -> build\lib.win-amd64-cpython-37\bmtrain\inspect
copying bmtrain\inspect\model.py -> build\lib.win-amd64-cpython-37\bmtrain\inspect
copying bmtrain\inspect\tensor.py -> build\lib.win-amd64-cpython-37\bmtrain\inspect
copying bmtrain\inspect\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain\inspect
creating build\lib.win-amd64-cpython-37\bmtrain\loss
copying bmtrain\loss\cross_entropy.py -> build\lib.win-amd64-cpython-37\bmtrain\loss
copying bmtrain\loss\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain\loss
creating build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
copying bmtrain\lr_scheduler\cosine.py -> build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
copying bmtrain\lr_scheduler\exponential.py -> build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
copying bmtrain\lr_scheduler\linear.py -> build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
copying bmtrain\lr_scheduler\noam.py -> build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
copying bmtrain\lr_scheduler\no_decay.py -> build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
copying bmtrain\lr_scheduler\warmup.py -> build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
copying bmtrain\lr_scheduler\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain\lr_scheduler
creating build\lib.win-amd64-cpython-37\bmtrain\nccl
copying bmtrain\nccl\enums.py -> build\lib.win-amd64-cpython-37\bmtrain\nccl
copying bmtrain\nccl\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain\nccl
creating build\lib.win-amd64-cpython-37\bmtrain\optim
copying bmtrain\optim\adam.py -> build\lib.win-amd64-cpython-37\bmtrain\optim
copying bmtrain\optim\adam_offload.py -> build\lib.win-amd64-cpython-37\bmtrain\optim
copying bmtrain\optim\optim_manager.py -> build\lib.win-amd64-cpython-37\bmtrain\optim
copying bmtrain\optim\__init__.py -> build\lib.win-amd64-cpython-37\bmtrain\optim
running build_ext
error: [WinError 5] 拒绝访问。

**Comment by a710128** [comment]
Probably because of insufficient permissions. Here is the solution. [stackoverflow](https://stackoverflow.com/questions/32167418/python-pip-install-trouble-shooting-permissionerror-winerror-5-access-is)
